<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot selection</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="rawTab">
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QWidget" name="normalizedTab">
      <attribute name="title">
       <string>Normalized</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QWidget" name="decomposedTab">
      <attribute name="title">
       <string>Decomposed</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Frequency Bands</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="deltaCB">
            <property name="text">
             <string>delta</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="thetaCB">
            <property name="text">
             <string>theta</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="alphaCB">
            <property name="text">
             <string>alpha</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="betaCB">
            <property name="text">
             <string>beta</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="averagePowerBandTab">
      <attribute name="title">
       <string>Average Band Power</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="QWidget" name="featuresTab">
      <attribute name="title">
       <string>Other Features</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="hfdCB">
            <property name="text">
             <string>Higuchi Fractal Dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pfdCB">
            <property name="text">
             <string>Petrosian Fractal Dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthActivityCB">
            <property name="text">
             <string>Hjorth Activity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthMobilityCB">
            <property name="text">
             <string>Hjorth Mobility</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthComplexityCB">
            <property name="text">
             <string>Hjorth Complexity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="engagementCB">
            <property name="toolTip">
             <string>This feature doesn't use an specific channel</string>
            </property>
            <property name="text">
             <string>Engagement Level</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameTB">
     <property name="toolTip">
      <string>Window name</string>
     </property>
     <property name="statusTip">
      <string>Window name</string>
     </property>
     <property name="whatsThis">
      <string>Window name</string>
     </property>
     <property name="placeholderText">
      <string>Window name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="apButton">
     <property name="text">
      <string>Add plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
